### PR TITLE
New doc page for Essentia TensorFlow models

### PR DESCRIPTION
**Pull request by dbogdanov**
New page with a complete overview of all available models.

### CODE REVIEW

File: doc/sphinxdoc/models.rst [hunk 1]
```diff
@@ -0,0 +1,653 @@
...
+* ``openl3-music-mel256-emb512``
+* ``openl3-music-mel256-emb6144``
+
+
+AudioSet-VGGish
+---------------
+
+Audio embeddings model accompanying the AudioSet dataset, trained to predict tags from Youtube videos.
+
+Dataset: preliminary subset of Youtube-8M.
```
Comment (dbogdanov) [inline doc/sphinxdoc/models.rst:581]: Simplify: "Subset of Youtube-8M"?

File: doc/sphinxdoc/models.rst [hunk 2]
```diff
@@ -0,0 +1,653 @@
...
+* ``openl3-music-mel128-emb512``
+* ``openl3-music-mel128-emb6144``
+* ``openl3-music-mel256-emb512``
+* ``openl3-music-mel256-emb6144``
+
+
+AudioSet-VGGish
+---------------
+
+Audio embeddings model accompanying the AudioSet dataset, trained to predict tags from Youtube videos.
```
Comment (dbogdanov) [inline doc/sphinxdoc/models.rst:579]: For consistency with OpenL3: "Audio embeddings model accompanying the AudioSet dataset trained in a supervised manner using tag information for YouTube videos." ?

File: doc/sphinxdoc/models.rst [hunk 3]
```diff
@@ -0,0 +1,653 @@
...
+
+Music classification by mood (5 mood clusters):
+
+`1: passionate, rousing, confident, boisterous, rowdy`,
+`2: rollicking, cheerful, fun, sweet, amiable/good natured`,
+`3: literate, poignant, wistful, bittersweet, autumnal, brooding`,
+`4: humorous, silly, campy, quirky, whimsical, witty, wry`,
+`5: aggressive, fiery, tense/anxious, intense, volatile, visceral`
+
+Dataset: MIREX Audio Mood Classification Dataset
```
Comment (dbogdanov) [inline doc/sphinxdoc/models.rst:500]: missing "."

File: doc/sphinxdoc/models.rst [hunk 4]
```diff
@@ -0,0 +1,653 @@
...
+This is a list of pre-trained TensorFlow models available in Essentia for various (music) audio analysis and classification tasks. To use the models in Essentia, see the `machine learning inference <machine_learning.html>`_ page.
+
+
+All the models created by the MTG are licensed under CC BY-NC-SA 4.0 (https://creativecommons.org/licenses/by-nc-sa/4.0/) and are also available under proprietary license upon request (https://www.upf.edu/web/mtg/contact).
+
+
+The models are serialized in `Protocol Buffer <https://developers.google.com/protocol-buffers/>`_ files suitable for inference with Essentia and TensorFlow. Additionally, some models are also available in `TensorFlow.js <https://www.tensorflow.org/js/models>`_) format. Each ``.pb`` file is coupled with a ``.json`` file containing its metadata.
+
+
+Model file naming conventions:
```
Comment (palonso) [inline doc/sphinxdoc/models.rst:15]: I would remove this section or move each line to its respective section

File: doc/sphinxdoc/models.rst [hunk 5]
```diff
@@ -0,0 +1,653 @@
+.. Essentia TensorFlow models
+
+Essentia TensorFlow models
```
Comment (palonso) [inline doc/sphinxdoc/models.rst:3]: The page ends being quite large. I think this introduction needs bullet points with the name of the sections with hyperlinks

File: doc/sphinxdoc/models.rst [hunk 6]
```diff
@@ -0,0 +1,653 @@
...
+==========================
+
+This is a list of pre-trained TensorFlow models available in Essentia for various (music) audio analysis and classification tasks. To use the models in Essentia, see the `machine learning inference <machine_learning.html>`_ page.
+
+
+All the models created by the MTG are licensed under CC BY-NC-SA 4.0 (https://creativecommons.org/licenses/by-nc-sa/4.0/) and are also available under proprietary license upon request (https://www.upf.edu/web/mtg/contact).
```
Comment (palonso) [inline doc/sphinxdoc/models.rst:9]: created -> trained?

File: doc/sphinxdoc/models.rst [hunk 7]
```diff
@@ -0,0 +1,653 @@
...
+- ``<task>`` and ``<source-task>``: the dataset in which the model, or the source model was trained.
+- ``<target-task>``: the dataset in which the target model was trained (only for the transfer learning classifiers).
+- ``params``: parameters specific to a particular family of models.
+- ``version``: represents the incremental version of the model. We do not guaranty full compatibility between different versions.
+
+
+
+Audio event recognition
+^^^^^^^^^^^^^^^^^^^^^^^
+Download: https://essentia.upf.edu/models/audio-event-recognition/
```
Comment (palonso) [inline doc/sphinxdoc/models.rst:38]: Minor thing: I don't like much how this looks.
[Download](https://essentia.upf.edu/models/audio-event-recognition/)
or..

[See all the models](https://essentia.upf.edu/models/audio-event-recognition/)

File: doc/sphinxdoc/models.rst [hunk 8]
```diff
@@ -0,0 +1,653 @@
...
+
+Dataset: AudioSet.
+
+Output: activations.
+
+This model is useful for audio embeddings.
+
+Models:
+
+* ``audioset-yamnet``
```
Comment (palonso) [inline doc/sphinxdoc/models.rst:155]: Hyperlink to the latest version of the model instead of the names?

**Review by palonso — CHANGES_REQUESTED** [review]

File: doc/sphinxdoc/models.rst (hunk 7 above)
Comment (dbogdanov) [reply doc/sphinxdoc/models.rst:38]: "Download" is probably too short to be easily noticed. "See all the models" would suggest there is more info about the models than on this page.

What about "Download model files" or "Download models"?

File: doc/sphinxdoc/models.rst (hunk 8 above)
Comment (dbogdanov) [reply doc/sphinxdoc/models.rst:155]: Maybe some does not necessarily want the last version but access to all version? Also (in general, not related to this model) we have separate files for TensorFlow.js so this won't work well for all use-cases.

File: doc/sphinxdoc/models.rst (hunk 6 above)
Comment (dbogdanov) [reply doc/sphinxdoc/models.rst:9]: This does not give additional details on the origin of the models.

File: doc/sphinxdoc/models.rst (hunk 7 above)
Comment (palonso) [reply doc/sphinxdoc/models.rst:38]: I like Download model files

**Pull request merged**